src/Microsoft.TestPlatform.ObjectModel/TestCase.cs

```diff
@@ -165,34 +104,6 @@ public Guid Id
             }
         }
 
-        /// <summary>
-        /// Gets or sets the fully specified type name metadata format.
-        /// </summary>
-        /// <example>
-        ///     <code>NamespaceA.NamespaceB.ClassName`1+InnerClass`2</code>
-        /// </example>
-        [DataMember]
-        public string ManagedType { 
```

Comment:
Does the serializer in TP work well with deleted members?

Reply:
These members don't exist on older releases, so removed to keep compatibility.